ports/azure_sdk.md

```diff
@@ -0,0 +1,341 @@
+
+  ```
+  ./scripts/ia32-generic-qemu-net.sh
+  ```
+
+ 3. Configure network and run `ash` (Busybox applet) using `rc` script:
+
+  - Note: `IP` is assigned using `DHCP`, there are also other possibilities, for more please check the configuration file located in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc.conf.d/network`
+
+  - Note: There are also executed other programs, for more information please check the content of the `rc` file for `ia32-generic-qemu` in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc`
+
+  ```
+  /linuxrc
+  ```
+
+  - As You can see, the advanced version of `Phoenix-RTOS` with `POSIX` shell has been started:
+
+  <img src="_images/azure_linuxrc.png" width="700px">
+
+  - Now You can check the internet connection using the `ping` applet.
+
+  - To make the configuration complete please set the current date using the `date` applet, like below:
+
+  <img src="_images/azure_date.png" width="700px">
+
+  - Note: The required argument entered after `@` is EPOCH (POSIX time format) - You can gt it for example from [here](https://www.epochconverter.com/).
+
+
+
+### Running the IoThub Client sample
+
+Providing the internet connection is working and the current date is set, You can run the built `iothub_ll_telemetry_sample`:
+
+```
+/bin/iothub_ll_telemetry_sample
+```
+
+<img src="_images/azure_sample.png" width="700px">
+
+ - Note: The entered connection string was added to the sample's source code, You can find it in `_build/ia32-generic-qemu/azure_sdk/azure-iot-sdk-c-lts_01_2022/iothub_client/samples/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.c`.
+
+You can read the sent messages from Azure, for example using `AzureIotHub VS Code Extension` by clicking `Start Monitoring Built-in Event Endpoint`:
+
+ ```
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+```
+
+## Using azure-iot-sdk-c
+
+The above guide only shows how to run one of the provided samples, if You want to write Your own programs using the sdk please follow the next instructions. It may be helpful for the other architectures, like `armv7m7-imxrt106x-evk`, where the generated sample may not work. That's the reason why the following example is adjusted to the configuration with `mbedtls` intended for 'smaller' targets (now only the `imxrt106x` is supported). If You want to write Your own programs intented for `openssl` configuration ('larger' targets, like `ia32-generic-qemu`) there will be a few differences, like no need for setting the certificate in code and increasing stack size in Makefile.
```

Comment:
```suggestion
The above guide shows how to run only one of the provided samples. To write your own programs using the SDK, please read the following instructions. It may be helpful for other architectures, like `armv7m7-imxrt106x-evk`, where the previously generated sample may not work. That's the reason, why the following example is adjusted to the configuration with `mbedtls` intended for 'smaller' targets (now only the `imxrt106x` is supported). If you want to write your own programs intended for the `openssl` configuration ('larger' targets, like `ia32-generic-qemu`) there are few differences, e.g. there is no need for setting the certificate in code and increasing the stack size in Makefile.
```